### ISSUE TITLE
Display Multiple Pets In Organized Fashion

### ISSUE THREAD

**Issue by dinaolmelak** [Techwise-Team-15/my_pet#18]
# Feature Request

**Description:**
The pet selection currently just shows one pet. The main goal of this ticket will be to display multiple pets on one screen in an organized fashion.

**Motivation:**
For our First Release milestone, it is essential to complete this ticket so that the user could select a specific pet.

**Current Behavior:**
The file currently displays all three pets but they are not lined up in a row.  The pets are also not playing their specific numbers of frames. 

**Proposed Solution:**
In order to display multiple pets, we will need to have all the specifications and animations set for the other pets as we did for the Raccoon pet. After that on the pet selection, we can just call and specify the locations where to display those pets.

**Code Snippet:**
N/A yet

**Additional Information:**
So basically we want to see multiple pets when opening up the `pet_selection.py`. Please  complete #19 before performing this task.

**Priority:**
- [ ] Low
- [ ] Medium
- [x] High

**Comment by RuthAnn1** [comment]
Successfully added pet frames.  You can check the pull request that is linked.
